processes/credential_exposure.md

```diff
@@ -0,0 +1,53 @@
+# Credential exposure
+
+Credentials are things like passwords or API keys. They are any values you wouldn't want exposed to the public but which your app or service needs to do its job.
+
+We haven't settled on a specific standard for how we handle them e.g. [12 factor app](https://12factor.net/config). But the one thing we are all agreed on is that they should not be committed with your project's source code.
+
+Yet mistakes happen and sometimes these secrets become exposed.
+
+This process will take you through what to do if that happens. The key point is
+
+> once out *always* consider your secret as compromised
+
+No matter the speed with which you rectify the situation, or how little you think the chance is for people to see it. Treat it as compromised.
+
+## Contact web-ops
+
+The first step is to call [David Blackburn](https://github.com/davidblackburn) or [Tom Tant](https://github.com/TTEA1990).
+
+We cannot post contact numbers here. But they are available using our internal systems.
+
+Follow this up with an email that contains a link to the commit, and explains what was exposed.
+
+## Fix your git
+
+Next is fix your git tree. This involves removing the offending commit from your git history.
+
+### On a branch
+
+If the commit was on a branch the fix involves amending or removing the commit. You can use git commands such as `git commit --amend` or `git rebase -i` to amend the current branches history.
+
+You will then follow this with a `git push -f`.
+
+### On master
+
+If it was the last commit on master, you can get away a `git commit --amend` and a `git push -f` if you have the rights.
+
+Else you will need to follow a much more involved strategy. GitHub provide a [guide](https://help.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository). If you are unsure contact the development community or speak to our principle developers.
```

Comment:
principal not principle

Reply:
I think the document needs to morph or be split to reflect the use of platforms other than AWS.  While a lot of content is common, things like Webops contacts are not.  I'll see what I can find out about contacts for Azure.